### DISCUSSION TITLE
Unauthenticated request to APPSEC

### DISCUSSION THREAD

**Discussion by AlexanderShK** [ZoeyVid/NPMplus#1281] [Q&A] [answered]
Hello,
I'm trying to setup multi-server configuration with https connection between machines. Right now there two VMs (LAPI server and NPMplus+Firewall Bouncer). Everything works as intended until https is used. With https in-use AppSec stops working with error `AppSecCheck(): Unauthenticated request to APPSEC`
Sorry for bad English 😅

**Comment by Zoey2936** [comment]
what do you mean with https is used?

**Reply by AlexanderShK** [reply, marked as answer]
I mean this - https://www.crowdsec.net/blog/secure-a-multi-server-installation-with-https
But anyway, I figured it out, I forgot to specify the SAN in the self signed certificate. And in all other modules it was ignored with `insecure_skip_verify: true`
Sorry to bother you 😅
